doc/configuration/outgoing-connections.md

```diff
@@ -446,3 +446,10 @@ Enables SNI extension to TLS protocol. If set to `true`, the `server_name_indica
 * **Example:** `tls.server_name_indication_host = "domain.com"`
 
 Domain against which the certificates will be checked, using SNI. It can be specified only when `server_name_indication` is set to `true`.
+
+### `outgoing_pools.*.*.connection.tls.server_name_indication_protocol`
+* **Syntax:** string, one of "default" or "https"
+* **Default:** not set
```

Comment:
I hate how a literal "default" option is not set by default. And from what I see setting it to "default" is the same as not setting it, so maybe skip the "default" possibility altogether?

Reply:
More or less fixed, and also fixed testing coverage.